FilePersistence/test/VersionControlMergetests.cpp

```diff
@@ -163,12 +163,69 @@ class FILEPERSISTENCE_API TwoChangesDifferentLists
 	Signature sig;
 	sig.name_ = "Chuck TESTa";
 	sig.eMail_ = "chuck@mergetest.com";
-	auto tree = merge->mergedTree();
-	merge->commit(sig, sig, "This is the result of merge test \"TwoChangesDifferentLists\"");
-	CHECK_CONDITION(!merge->hasConflicts());
-	CHECK_CONDITION(!tree->find(QUuid{"{00000000-0000-0000-0000-000000000210}"}));
-	CHECK_CONDITION(tree->find(QUuid{"{00000000-0000-0000-0000-000000000408}"}));
+	if (!merge->isAlreadyMerged())
```

Comment:
Ok, we don't actuall need to check this. In this test, we know that something either happened or it didn't. In this case I guess it was not already merged.

Either way, instead of having an `if`, just put a `Q_ASSERT` that the respective state holds and then just go on checking different conditions.

The same applies for all the other tests.